google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/application/AppEngineApplicationCreateDialog.java

```diff
@@ -147,8 +147,8 @@ private void trackApplicationCreateFailure() {
         .ping();
   }
 
-  private void setStatusMessageAsync(final String message, final boolean isError) {
```

Comment:
errorprone doesn't, but the IJ linter does, so I thought I'd remove it. I was wondering if this method had been called before actually using that argument, but then that call was deleted. If there aren't any plans to start using this with a variable isError, then I'd like to delete based on YAGNI.

`setStatusMessage()` is called with variable isError, which is why I didn't cut it there as well.